docs/content/documentation/systems/Functions/framework/CompositeFunction.md

```diff
@@ -1,8 +1,13 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
+# Composite
 
-# CompositeFunction
 !description /Functions/CompositeFunction
 
+The `CompositeFunction` type takes an arbitrary set of functions, provided in
+the `functions` parameter, evaluates each of them at the appropriate time
+and position, and multiplies them together.  The function can optionally be
+multiplied by a scale factor, which is specified using the `scale_factor`
+parameter.
+
 !parameters /Functions/CompositeFunction
```

Comment:
I would like to see each one of these have some example input syntax, see what I did in the Markers.